### DISCUSSION TITLE
How to build the doc of typst?

### DISCUSSION THREAD

**Discussion by Freed-Wu** [typst/typst#1324] [Q&A] [answered]
<https://github.com/typst/typst/tree/main/docs> contains markdown, yaml and rust. How to build it to get the doc of typst?

**Comment by Enivex** [comment, marked as answer]
Not possible currently, as it's closed source as far as I know, but it's planned.

**Reply by Freed-Wu** [reply]
Thanks, the possibility to build the document is helpful for providing a translation for typst.
